### ISSUE TITLE
Add flag for "isSubscribable"

### ISSUE THREAD

**Issue by collinbarrett** [collinbarrett/FilterLists#28]
Add a flag property for if a list is directly "subscribe-able". If it is a .zip file, etc., this flag will indicate that the "Add" button should not be displayed.

Instead of a property, this could be something that is automated as well. (i.e., if file is a .txt vs a .zip, etc.). That would need extensive testing/debugging.

Feature overlap/similar to #7

**Comment by collinbarrett** [comment]
Syntax IDs 7 (uMatrix/uBlock Origin Dynamic) and maybe 10 (TPL / IE) should always hide subscribe buttons.

**Comment by elypter** [comment]
i read that nano adblocker is planing to support compressed lists but its not implemented yet afaik however compressed files could become subscribable for some adblockers.

**Comment by collinbarrett** [comment]
yeah, that's a good point. might put an optional dropdown control that allows users to select which software they use. then, hide subscribe buttons known to not be supported by that software.

**Comment by collinbarrett** [comment]
Added `isAbpSubscribable` to [wiki](https://github.com/collinbarrett/FilterLists/wiki/Software).

**Comment by collinbarrett** [comment]
Thanks, @DandelionSprout for updating some of these in #489 . Removing help wanted as I think that probably covers the initial set of support for `abp` subscribe links.

**Comment by gwarser** [comment]
> We should hide subscribe buttons if the `viewUrl` is not plain-text (e.g. `.7z`, `.zip`)

`.tar.gz` should be added to the list.

Plain text, gzip compressed may be fine (needs testing).

**Comment by DandelionSprout** [comment]
I actually came close to creating a pull a week ago that'd do just that for all three of those formats. But two problems got in my way.

1) Currently, if the URL for a compressed file is in ViewUrlMirror1 or ViewUrlMirror2, there would be no way to access the link without using the F12 menu.

2) I additionally wanted to re-enable TPL subscription links so that they simply lead to https://raw.githack.com/collinbarrett/FilterLists/master/data/TPLSubscriptionAssistant.html, and in turn I wanted to give those links a light blue colour instead of dark blue. But I couldn't figure out how to add colour hex values to https://github.com/collinbarrett/FilterLists/blob/master/web/src/components/SubscribeButtons.tsx.

An alternative for №1 could be to re-colour such links orange, and/or change the "message" value to "Download this list in compressed format". But that'd hinge on figuring out a solution for №2, which could then be used for this as well.
